### PR TITLE
Add mouse click handler

### PR DESCRIPTION
**Pull request by ucay**
Issue : #4 
Add basic mouse click handler to results template.
If wanna use onTap instead of onClick, just create custom result template.

```js
<FuzzySearch
  list={list}
  keys={['author', 'title']}
  width={430}
  onSelect={action('selected')}
  resultsTemplate={(props, state, styles, clickHandler) => {
    return state.results.map((val, i) => {
      const style = state.selectedIndex === i ? styles.selectedResultStyle : styles.resultsStyle;
      return (
        <div
          key={i}
          style={style}
          {/*onClick={() => clickHandler(i)}*/}
          onTap={() => clickHandler(i)}
        >
          {val.title}
          <span style={{ float: 'right', opacity: 0.5 }}>by {val.author}</span>
        </div>
      );
    });
  }}
/>
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1 @@
...
-shouldSort| true | Whether to sort the result list, by score.
-sortFn|`Array.prototype.sort`|The function that is used for sorting the result list.
-threshold|0.6|At what point does the match algorithm give up. A threshold of `0.0` requires a perfect match (of both letters and location), a threshold of `1.0` would match anything.
-tokenize|false|When true, the search algorithm will search individual words and the full string, computing the final score as a function of both. Note that when tokenize is true, the `threshold`, `distance`, and `location` are inconsequential for individual tokens.
-verbose|false|Will print to the console. Useful for debugging.
-
-##License
-MIT @ Ritesh Kumar
-
+This is for temporary personal use only, please use https://github.com/ritz078/react-fuzzy-search
```
Comment (ritz078) [inline README.md]: why have you done this ?

File: package.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {
-  "name": "react-fuzzy",
+  "name": "ucay-react-fuzzy",
```
Comment (ritz078) [inline package.json]: same. you should leave this as it is.

**Comment by ucay** [comment]
I'm sorry.
My mistake forgot to create a new branch after PR.
I've cleaned it up.

**Comment by ritz078** [comment]
@elghali @ucay 

I was away from github due to health reasons. I generally use to reply ASAP but recently I have been busy with health problems. Sorry again. I will try to do these soon. If anyone of you wants to be a contributor, I can add you so that project runs smoothly even when I am not available.

**Comment by ritz078** [comment]
@ucay If you can resolve conflicts else I will do it tonight and merge. Thanks for the PR. 👍

**Comment by elghali** [comment]
Oh I'm sorry, wish you got better, sure I'd be glad to help

**Comment by ritz078** [comment]
@elghali I have added you as the collaborator.

**Pull request merged**